### ISSUE TITLE
Unable to interpolate Integers?

### ISSUE THREAD

**Issue by peterfigure** [uchuhimo/konf#57]
given

```
database {
  name = "servicedb"
  hostname = "db"
  port = 5432
}
flyway {
  url= "jdbc:postgresql://${database.hostname}:${database.port:-5432}/${database.name}"
}
```

it fails with

```
Caused by: com.uchuhimo.konf.source.WrongTypeException: source 5432 in [type: HOCON, url: file:/xxx/server/build/resources/main/application.conf, resource: application.conf] has type Integer rather than String
	at com.uchuhimo.konf.source.SourceKt.asValueOf(Source.kt:448) ~[konf-core-0.22.1.jar:na]
	at com.uchuhimo.konf.source.TreeLookup.lookup(Source.kt:407) ~[konf-core-0.22.1.jar:na]
	at org.apache.commons.text.lookup.InterpolatorStringLookup.lookup(InterpolatorStringLookup.java:151) ~[commons-text-1.8.jar:1.8]
```

is it possible to do this?

**Comment by peterfigure** [comment]
a workaround is to define the value in the HOCON conf as a string

**Comment by uchuhimo** [comment]
@peterfigure I have released a new version v0.23.0 that support to interpolate Byte/Short/Int/Long/BigInteger/Char. You can try it now. Any further feedback is welcome.
